documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -359,8 +246,7 @@ Model In Image model files, application archives, and the WDT installation files
        $ unzip /tmp/mii-sample/model-images/weblogic-deploy.zip -d .
        $ rm ./weblogic-deploy/bin/*.cmd
        ```
-       In a later step, we will specify a domain resource `domain.spec.configuration.model.wdtInstallHome`
-         attribute that references this WDT installation directory.
+       The optional `domain.spec.configuration.model.auxiliaryImages[].sourceWDTInstallHome` attribute can be configured to this reference this WDT installation directory.
```

Comment:
This new sentence "The optional ..." looks out of place. Does it belong here? Should it be simply deleted?

Reply:
I delete it.